tests/cpu/transforms/LookTransform_tests.cpp

```diff
@@ -559,3 +562,191 @@ environment: { FILE1: cdl_test1.cc, FILE2: cdl_test1.cc }
     OCIO_CHECK_EQUAL(std::string("FILE2"), usedContextVars->getStringVarNameByIndex(1));
     OCIO_CHECK_EQUAL(std::string("cdl_test1.cc"), usedContextVars->getStringVarByIndex(1));}
 
+
+OCIO_ADD_TEST(LookTransform, inline_look_transform)
+{
+    constexpr const char * OCIO_CONFIG{ R"(
+ocio_profile_version: 2
+
+search_path: luts
+
+roles:
+  default: raw
+
+displays:
+  sRGB:
+    - !<View> {name: Raw, colorspace: raw}
+
+looks:
+  - !<Look>
+    name: look1
+    process_space: log
+    transform: !<CDLTransform> {sat: 0.8}
+
+colorspaces:
+  - !<ColorSpace>
+    name: raw
+    family: raw
+    bitdepth: 32f
+    isdata: false
+
+  - !<ColorSpace>
+    name: log
+    to_reference: !<LogTransform> {base: 2, direction: inverse}
+
+  - !<ColorSpace>
+    name: vd
+    from_reference: !<ExponentTransform> {value: [2.4, 2.4, 2.4, 1], direction: inverse}
+
+  - !<ColorSpace>
+    name: vd_graded
+    from_reference: !<GroupTransform>
```

Comment:
I think that the `GroupTransform` level does not add any useful information to the unit test so it can be removed to simplify the config. The same comment applies below.

Reply:
Good point, done